modules/customize-certificates-api-add-named.adoc

```diff
@@ -78,3 +78,21 @@ spec:
         name: <secret>
 ...
 ----
+
+. Check that a new revision of the Kubernetes API server pods has rolled out. This will take several minutes.
++
+[source,terminal]
+----
+$ oc get clusteroperators kube-apiserver
+----
++
+Do not continue to the next step until `PROGRESSING` is listed as `False`, as shown in the following output:
++
+.Example output
+[source,terminal]
+----
+NAME             VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE
+kube-apiserver   4.7.0     True        False         False      145m
```

Comment:
s/4.7.0/{product-version}.0/ if you want quicker backports

Reply:
I have added the version variable.